tao_compiler/mlir/disc/transforms/disc_lower_to_library_call.cc

```diff
@@ -104,6 +104,12 @@ void InsertSyncOnStream(Operation* op, Value ctx, Value stream_handle,
                               "sync_on_stream", false, "gpu");
 }
 
+// return true if op in fusion block
+bool isInFusionBlock(Operation* op) {
```

Comment:
Nit: move to `fusion_utils.cc`

Reply:
this function is only used in `disc-lower-to-library-call-pass`, maybe here is okay, let's move to fusion_utils.cc if it's a common function.